### PR TITLE
update mineru 1.5.4 doc

### PR DESCRIPTION
**Pull request by yingjie-han**
add pipeline backend doc

### CODE REVIEW

**Review by Wei-Lin-Intel — APPROVED** [review]
LGTM

File: scripts/docs/MinerU2.5.4Gaudi部署指南.md [hunk 1]
```diff
@@ -1,62 +1,44 @@
...
-部署minerU vllm gaudi server
-部署minerU api 服务
-minerU CLI和API 访问方式参考用例
+本指南提供在 MinerU v2.5.4 上使用 Intel Gaudi 作为硬件加速器通过 pipeline后端/VLLM 后端进行部署的详细步骤。
 
 ## 前提条件
 
 - 已安装 Intel Gaudi 软件栈
 - 支持 Gaudi 对应驱动版本的基础Docker镜像
+可参考[《https://github.com/HabanaAI/vllm-fork/blob/aice/v1.21.0/scripts/docs/vllm%E6%8E%A8%E7%90%86%E6%89%8B%E5%86%8C.md》]
```
Comment (czhu15) [inline scripts/docs/MinerU2.5.4Gaudi部署指南.md]: 1.21.0 ==> 1.22.0

File: scripts/docs/MinerU2.5.4Gaudi部署指南.md [hunk 2]
```diff
@@ -1,62 +1,44 @@
...
+为容器设置正确的网络设置，确保容器可以正常访问 github。
 
 ```bash
-cd /data/vllm-fork
- VLLM_TARGET_DEVICE=hpu pip install .
-
-cd /data/MinerU
-pip install .[core] -i https://mirrors.aliyun.com/pypi/simple
-
+git clone https://github.com/yingjie-han/MinerU.git
```
Comment (czhu15) [inline scripts/docs/MinerU2.5.4Gaudi部署指南.md]: "https://github.com/yingjie-han" do we use the personal github? @Wei-Lin-Intel

File: scripts/docs/MinerU2.5.4Gaudi部署指南.md [hunk 3]
```diff
@@ -68,7 +50,20 @@ https://github.com/opendatalab/MinerU/blob/master/mineru.template.json
...
-### 1. 环境变量
+### 4. 使用MinerU vlm-vllm-engine backend
+
+#### 4.1 为vlm-vllm-engine backend安装vllm
+
+```bash
+git clone https://github.com/HabanaAI/vllm-fork.git -b aice/v1.22.0
+pip install -e . -i https://mirrors.aliyun.com/pypi/simple
+git clone https://github.com/vllm-project/vllm.git
+cp -r vllm/vllm/v1/sample/logits_processor vllm-fork/vllm/v1/sample/logits_processor
```
Comment (czhu15) [inline scripts/docs/MinerU2.5.4Gaudi部署指南.md]: "cp -r vllm/vllm/v1/sample/logits_processor vllm-fork/vllm/v1/sample/logits_processor"
pls check with Tina to confirm if above cp is still needed.

**Review by czhu15 — APPROVED** [review]
LGTM except some minor comments.
Pls ask Tina for final check.

**Comment by czhu15** [comment]
@tinafengfun, can you review this doc? I will merge it if you are OK.

File: scripts/docs/MinerU2.5.4Gaudi部署指南.md [hunk 4]
```diff
@@ -1,62 +1,44 @@
...
-## 环境准备
+## 使用MinerU v2.5.4
 
-### 1. 获取gaudi vllm 源代码和MinerU 源代码
-
-```bash
-git clone https://github.com/HabanaAI/vllm-fork.git -b aice/v1.22.0
-git clone https://github.com/opendatalab/MinerU.git
-cd MinerU
-git checkout release-2.5.4
```
Comment (tinafengfun) [inline scripts/docs/MinerU2.5.4Gaudi部署指南.md]: Could please keep this part as vlm is using MinerU public repo and add private repo to pipeline section.

File: scripts/docs/MinerU2.5.4Gaudi部署指南.md (hunk 2 above)
Comment (tinafengfun) [reply scripts/docs/MinerU2.5.4Gaudi部署指南.md]: vlm backend deployment use public repo but I do think pipeline may use private repo. So Could please separate and specify the two repo for two usage?

File: scripts/docs/MinerU2.5.4Gaudi部署指南.md (hunk 3 above)
Comment (tinafengfun) [reply scripts/docs/MinerU2.5.4Gaudi部署指南.md]: This is not demanding as we use export VLLM_USE_V1=0 to disable v1 logits processor for this usage. If customer perfer to use minerU's own sampler , they could handle this from command line and hack the code by referring to minerU's instruction

File: scripts/docs/MinerU2.5.4Gaudi部署指南.md (hunk 1 above)
Comment (yingjie-han) [reply scripts/docs/MinerU2.5.4Gaudi部署指南.md]: As there is no 1.22 doc now,   I removed 1.21 link.

File: scripts/docs/MinerU2.5.4Gaudi部署指南.md (hunk 2 above)
Comment (yingjie-han) [reply scripts/docs/MinerU2.5.4Gaudi部署指南.md]: I changed the repo to gitee, thanks.

**Comment by tinafengfun** [comment]
LGFM

**Comment by czhu15** [comment]
@yingjie-han , pls fix the pre-commit issues.

**Comment by yingjie-han** [comment]
> @yingjie-han , pls fix the pre-commit issues.

@czhu15 fixed.
